intelmq/bin/intelmqctl.py

```
@@ -653,7 +653,7 @@ def get_queues(self):
     def list_queues(self):
         source_queues, destination_queues, internal_queues, all_queues = self.get_queues()
         pipeline = PipelineFactory.create(self.parameters)
-        pipeline.set_queues(source_queues, "source")
+        pipeline.set_queues(None, "source")
```

We need to set at least a dummy value here, otherwise we would fail later.